config/routes.rb

```diff
@@ -1,10 +1,14 @@
-Rails.application.routes.draw do # rubocop:disable Metrics/BlockLength
+# rubocop:disable Metrics/BlockLength
```

Comment:
I think it would be best to disable this here https://github.com/alphagov/email-alert-api/blob/master/.rubocop.yml as that avoids the distraction of the comments in the file.

Whitehall might serve as a good example https://github.com/alphagov/whitehall/blob/master/.rubocop.yml

Reply:
Sure, thanks for the example!